packages/curvenote-cli/src/session/auth.ts

```diff
@@ -119,25 +123,51 @@ export async function setToken(log: Logger, token?: string) {
     ]);
     token = resp.token as string;
   }
-  const session = new Session(token);
+
+  log.debug('Creating session with new token');
+  const session = await Session.create(token);
```

Comment:
`Session`s constructor is private, `await Session.create()` is the only way to create one.